fastplotlib/widgets/image.py

```diff
@@ -927,9 +919,17 @@ def show(
             ImageWidget just uses the Gridplot output context
         """
         if self.gridplot.canvas.__class__.__name__ == "JupyterWgpuCanvas":
+            from ..layouts._frame._ipywidget_toolbar import (
```

Comment:
do these types of imports work with pyinstaller?

Reply:
Just as well as it did before this change ;) because they were in a conditional there as well. It might be a good idea to implement a pyinstaller hook like we [do in wgpu](https://github.com/pygfx/wgpu-py/blob/main/wgpu/__pyinstaller/hook-wgpu.py) to just include all modules.

Reply:
I made #434 for this, can look into it later.